Lib/fontTools/ttLib/ttFont.py

```diff
@@ -359,19 +431,21 @@ def _saveXML(
         writer.endtag("ttFont")
         writer.newline()
 
-    def _tableToXML(self, writer, tag, quiet=None, splitGlyphs=False):
+    def _tableToXML(self, writer: xmlWriter.XMLWriter, tag: str, quiet: bool | None = None, splitGlyphs: bool = False) -> None:
         if quiet is not None:
             deprecateArgument("quiet", "configure logging instead")
-        if tag in self:
-            table = self[tag]
-            report = "Dumping '%s' table..." % tag
-        else:
-            report = "No '%s' table found." % tag
-        log.info(report)
+
         if tag not in self:
+            report = "No '%s' table found." % tag
+            log.info(report)
```

Comment:
I realize this is (also) out of scope, but the recommended way of formatting logging calls would in this case look something like this:

```suggestion
            log.info( "No %r table found.", tag)
```

The `%r` effectively does a `repr(_)`, which gives you free quotes.
And to be honest I can't remember exactly why the format arguments should be passed like this, just that there's a good reason for doing so haha.